### ISSUE TITLE
Powershell will force close on any error

### ISSUE THREAD

**Issue by mddtest** [microsoft/winget-cli#1048]
# Brief description of your issue

Powershell will close when it encounters an error preventing you from seeing what error was created

# Steps to reproduce

run any winget app that fails and powershell will close
# Expected behavior

 powershell should stay open to see the error msg.

# Actual behavior

powershell force closes 

# Environment

```none
[winget --info]
Windows Package Manager v1.0.11451
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19042.985
Package: Microsoft.DesktopAppInstaller v1.11.11451.0


Any other software?
```

**Comment by denelon** [comment]
Are you executing this in Windows Terminal or a CMD session? 

If you run this via [Windows]+[r] the command completes and the session is closed.

**Comment by mddtest** [comment]
I was using Terminal, seems after windows update 21h1 it no longer happens.

**Comment by denelon** [comment]
I'll mark this as Area-External to keep it open for a while and see if this resolves the issue for others.

**Comment by denelon** [comment]
We've published the v1.1 [release candidate](https://github.com/microsoft/winget-cli/releases/tag/v1.1.12653) and updated our [troubleshooting guide](https://github.com/microsoft/winget-cli/tree/master/doc/troubleshooting).
